src/modules/infobox/components/styling/FeatureStyleEdit.vue

```diff
@@ -159,6 +167,55 @@ export default {
                 this.changeFeatureTitle({ feature: this.feature, title: value })
             },
         },
+        geometry: {
+            get() {
+                /*
+                Openlayers polygons coordinates are in a triple array
+                The first array is the "ring", the second is to hold the coordinates, which are in an array themselves
+                We don't have rings in this case, so we need to create an ol geometry
+                */
+
+                const geom = [this.feature.coordinates]
+
+                return new Polygon(geom)
+            },
+        },
+        isClosed: {
+            get() {
+                return (
+                    this.feature.coordinates[0][0] ===
+                        this.feature.coordinates[this.feature.coordinates.length - 1][0] &&
+                    this.feature.coordinates[0][1] ===
+                        this.feature.coordinates[this.feature.coordinates.length - 1][1]
```

Comment:
you might want to check that `this.feature.coordinates.length > 1` in order to say that it is closed, otherwise a single point will be considered as closed too